tango_simlib/tango_launcher.py

```diff
@@ -64,7 +64,7 @@ def start_device(opts):
     server_name = os.path.basename(opts.server_command)
     number_of_devices = len(opts.name)
     # Register tango devices if TANGO DB is being used
-    if not opts.file_name:
+    if opts.file_name:
```

Comment:
Comment and code is now misaligned because ` if opts.file_name ` indicates TANGO DB is NOT being used.

Reply:
As discussed in standup - the registration is still required for when TANGO DB is used too.